packages/uix-host-react/src/components/Extensible.tsx

```diff
@@ -77,33 +77,22 @@ export function Extensible({
   guestOptions,
   runtimeContainer,
   debug,
-  sharedContext = {},
+  sharedContext,
 }: PropsWithChildren<ExtensibleProps>) {
-  const [extensions, setExtensions] = useState({});
+  const hostName = appName || window.location.host || "mainframe";
 
+  const [extensions, setExtensions] = useState({});
   useEffect(() => {
     extensionsProvider()
-      .then((loadedExtensions: InstalledExtensions) => {
-        if (areExtensionsDifferent(extensions, loadedExtensions)) {
-          setExtensions(loadedExtensions);
```

Comment:
Here I'm using latest version of `extensions` state instead of one captured during callback creation.